README.md

```diff
@@ -1,53 +1,68 @@
-Gaffer
-======
+![Gaffer Logo](resources/GafferLogo.svg)
 
-Gaffer is an open source application framework designed specifically for creating tools for use in visual effects production. It builds on top of the Cortex libraries, adding a flexible node-based computation framework and a user interface framework for editing and viewing node graphs. Gaffer ships with a number of sample modules and applications, not least of which is a module for the on-demand generation of procedural scenes for rendering.
+# Gaffer #
 
-More information can be found on the [project homepage](http://gafferhq.org).
+Gaffer is an open source application framework designed for creating tools for use in VFX production. It builds on top of the Cortex libraries, adding a flexible node-based computation framework and a user interface framework for editing and viewing node graphs. Gaffer ships with a number of sample modules and applications, not least of which is a module for the on-demand generation of procedural scenes for rendering.
 
-Developer notes are available on the [project wiki](https://github.com/GafferHQ/gaffer/wiki).
+More information can be found on the [Gaffer homepage](https://gafferhq.org).
 
-Downloading
-===========
+Developer notes are available on the [Gaffer developer wiki](https://github.com/GafferHQ/gaffer/wiki).
+
+Participating in the Gaffer community requires abiding by the project's [Code of Conduct](https://github.com/GafferHQ/gaffer/blob/master/CODE_OF_CONDUCT.md).
+
+
+## Download ##
 
 Compiled binary releases are available for download from the [releases page](https://github.com/GafferHQ/gaffer/releases).
 
-Building
-========
+
+## Building ##
 
 [![Build Status](https://travis-ci.org/GafferHQ/gaffer.svg?branch=master)](https://travis-ci.org/GafferHQ/gaffer)
 
-Gaffer is a fairly large project, and as such has a fairly complex build process. Before you start, you'll want to make sure you have the following prerequisites installed on your system - these will be used to perform the build itself :
+Gaffer is a fairly large project, and as such has a fairly complex build process. Before you start, make sure you have the following prerequisites installed on your system, which will be used to perform the build itself:
 
-- [SCons](http://www.scons.org)
-- [Inkscape](http://inkscape.org)
-- [Sphinx](http://www.sphinx-doc.org/) (optional, minimum version 1.3)
+- [SCons](https://www.scons.org)
+- [Inkscape](https://inkscape.org)
+- [Sphinx](https://www.sphinx-doc.org/) 1.4 or higher (optional)
```

Comment:
I'm not sure higher works - I've locked it at 1.4 in GafferHQ/build, because of incompatibilities with recommonmark.